config/routes.rb

```diff
@@ -6,7 +6,7 @@
   
   resources :sessions, only: [:new, :create, :destroy]
   get '/login', to: 'sessions#new'
-  delete '/logout', to: 'sessions#destroy'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
This implementation is wrong.
Do you remember discussion with HTTP requests and actions?
What do you think LOGOUT's function is for?

It is a DELETE function. Not GET. Not POST. Not PUT. Not PATCH.

What else do you need to do if a function like LOGOUT or DELETE or REMOVE or PUT TO TRASH does?
You put **method: :delete** in VIEW page.

Review how to do Logout function and Delete function.